frontend/server/src/Controllers/School.php

```diff
@@ -360,8 +271,8 @@ public static function apiRank(\OmegaUp\Request $r) {
         $offset = is_null($r['offset']) ? 1 : intval($r['offset']);
         $rowCount = is_null($r['rowcount']) ? 100 : intval($r['rowcount']);
 
-        /** @var list<array{school_id: int, rank: int, score: float, name: string}> */
-        $rank = \OmegaUp\Cache::getFromCacheOrSet(
+        /** @var array{rank: list<array{school_id: int, country_id: int, rank: int, score: float, name: string}>, totalRows: int} */
+        $rankResult = \OmegaUp\Cache::getFromCacheOrSet(
```

Comment:
```suggestion
        return \OmegaUp\Cache::getFromCacheOrSet(
```

y aquí está la razón por la cual hice opcional lo del `'status' => 'ok'`! ya no es necesario hacer estos malabares nada más para agregar el `status`.